docs/getting_started/index.rst

```diff
@@ -1,11 +1,46 @@
 Getting started
 ===============
 
+Wagtail is built on the `Django web framework <https://www.djangoproject.com/>`_, so this document assumes you've already got the essentials installed. But if not, those essentials are:
+
+ * `Python <https://wiki.python.org/moin/BeginnersGuide/Download>`_
+ * `pip <https://pip.pypa.io/en/latest/installing.html>`_ (Note that pip is included by default with Python 2.7.9 and later and Python 3.4 and later)
+
+We'd also recommend:
+
+ * `Virtualenv <https://virtualenv.pypa.io/en/latest/installation.html>`_
+
+
+With the above installed and enabled, the quickest way to install Wagtail is::
+
+    pip install wagtail
+
+.. note::
+
+    Wagtail depends on the Pillow package for image manipulation. On some platforms Pillow requires additional steps to get images working in Wagtail. `More information here <http://pillow.readthedocs.org/en/latest/installation.html#os-x-installation>`_
```

Comment:
Could we just say "Install these two packages" instead of sending people to read a big long document?

But at least point to a more generic part of that doc (currently pointing to "OS X Installation" and not everyone uses OS X!)